src/lib/get-participations.ts

```diff
@@ -0,0 +1,20 @@
+import type { Participations } from '@guardian/ab-core';
+import { isObject, isString, storage } from '@guardian/libs';
+
+const participationsKey = 'gu.ab._participations';
+
+const isParticipations = (
+	participations: unknown,
+): participations is Participations => {
+	if (!isObject(participations)) return false;
+	return Object.values(participations).some((participation) => {
+		if (!isObject(participation)) return false;
+		if (!isString(participation.variant)) return false;
+		return true;
+	});
+};
+
+export const getParticipationsFromLocalStorage = (): Participations => {
```

Comment:
Minor:

I know we lifted this file from dotcom's `ab-localstorage.ts`. I prefer that naming for a couple of reasons:
- clear that this module is something to do with ab testing
- it is a (albeit) weak link to that module in DCR which will make it easier to track should we ever migrate this functionality to libs

Reply:
Yup agreed, I'll make the change